my-app/src/components/board.js

```diff
@@ -1,34 +1,11 @@
 import React from 'react';
 import Square from './square.js';
 
-class Board extends React.Component {
-    renderSquare(i) {
-      return <Square
-        value={this.props.squares[i]}
-        onClick={() => this.props.onClick(i)} />;
-    }
-  
-    render() {
-      return (
-        <div>
-          <div className="board-row">
-            {this.renderSquare(0)}
-            {this.renderSquare(1)}
-            {this.renderSquare(2)}
-          </div>
-          <div className="board-row">
-            {this.renderSquare(3)}
-            {this.renderSquare(4)}
-            {this.renderSquare(5)}
-          </div>
-          <div className="board-row">
-            {this.renderSquare(6)}
-            {this.renderSquare(7)}
-            {this.renderSquare(8)}
-          </div>
-        </div>
-      );
-    }
-  }
-
-  export default Board;
\ No newline at end of file
+const Board = ({ squares, onClick }) => (
+  <div className="board">
+    {squares.map((square, i) => (
+      <Square key={i} value={square} onClick={() => onClick(i)} />
```

Comment:
keep in mind that normally, giving index as a component key isn't a good practice. (https://kentcdodds.com/blog/understanding-reacts-key-prop)